flexget/components/trakt/trakt_lookup.py

```diff
@@ -61,51 +61,48 @@ def trakt_user_data_lookup(entry, field_name, data_type, media_type, lookup_func
     return entry
 
 
-class PluginTraktLookup:
-    """Retrieves trakt information for entries. Uses series_name,
-    series_season, series_episode from series plugin.
-
-    Example:
-
-    trakt_lookup: yes
+def _get_lookup_args(entry):
+    args = {
+        'title': entry.get('series_name', eval_lazy=False) or entry.get('title', eval_lazy=False),
+        'year': entry.get('year', eval_lazy=False) or entry.get('movie_year', eval_lazy=False),
+        'trakt_slug': (
+            entry.get('trakt_show_slug', eval_lazy=False)
+            or entry.get('trakt_movie_slug', eval_lazy=False)
+        ),
+        'tmdb_id': entry.get('tmdb_id', eval_lazy=False),
+        'tvdb_id': entry.get('tvdb_id', eval_lazy=False),
+        'imdb_id': entry.get('imdb_id', eval_lazy=False),
+        'tvrage_id': entry.get('tvrage_id', eval_lazy=False),
+    }
 
-    Primarily used for passing trakt information to other plugins.
-    Among these is the IMDB url for the series.
+    if entry.get('trakt_movie_id', eval_lazy=False):
+        args['trakt_id'] = entry['trakt_movie_id']
+    elif entry.get('trakt_show_id', eval_lazy=False):
+        args['trakt_id'] = entry['trakt_show_id']
+    elif is_movie(entry) and entry.get('trakt_movie_id', eval_lazy=True):
+        args['trakt_id'] = entry['trakt_movie_id']
+    elif entry.get('trakt_show_id', eval_lazy=True):
+        args['trakt_id'] = entry['trakt_show_id']
 
-    This information is provided (via entry):
-    series info:
-    trakt_series_name
-    trakt_series_runtime
-    trakt_series_first_aired_epoch
-    trakt_series_first_aired_iso
-    trakt_series_air_time
-    trakt_series_content_ratingi
-    trakt_series_genres
-    trakt_series_imdb_url
-    trakt_series_trakt_url
-    imdb_id
-    tvdb_id
-    trakt_series_actors
-    trakt_series_country
-    trakt_series_year
-    trakt_series_tvrage_id
-    trakt_series_status
-    trakt_series_overview
+    return args
 
-    trakt_ep_name
-    trakt_ep_season
-    trakt_ep_number
-    trakt_ep_overview
-    trakt_ep_first_aired_epoch
-    trakt_ep_first_aired_iso
-    trakt_ep_id
-    trakt_ep_tvdb_id
 
+def get_db_data_for(data_type, entry, session):
```

Comment:
Should this be in ./db.py ?

Reply:
Hmm. Not sure. Don't think db.py deals with entries at all right now. This is function is sortof about translating the entry into the db calls in db.py